lib/src/pageObjects/adminUIListScreen.js

```diff
@@ -1,110 +1,494 @@
+		 * 
+		 * @param {Object} fields The array of fields to click the value on.
+		 * @param {number} field.row The one-based row number of item the item containing the field under test.
+		 * @param {number} field.column The one-based column number of the field under test.
+		 * @param {Object} field.modelTestConfig The model test config that should be used for the field under test.
+		 * @param {Object} field.options Any options required by the field test object command clickListScreenFieldValue.
+		 */
+		clickItemFieldValue: function (fields) {
+			var browser = this;
+			if (fields) {
+				fields.forEach(function (field) {
+					var fieldSelector = getItemFieldSelector(field);
+					if (fieldSelector) {
+						var modelTestConfig = new field.modelTestConfig({ formSelector: fieldSelector });
+						if (modelTestConfig) {
+							var fieldTestObject = modelTestConfig[field.name];
+							if (fieldTestObject) {
+								if (fieldTestObject.commands && 'clickListScreenFieldValue' in fieldTestObject.commands) {
+									modelTestConfig[field.name].commands.clickListScreenFieldValue(browser, field.options);
+								} else {
+									throw new Error('clickItemFieldValue: clickListScreenFieldValue command not defined in ' + field.name + ' field test object!');
+								}
+							} else {
+								throw new Error("clickItemFieldValue: invalid field name!");
+							}
+						} else {
+							throw new Error("clickItemFieldValue: invalid field modelTestConfig!");
+						}
+					} else {
+						throw new Error("clickItemFieldValue: invalid field config row/column!");
+					}
+				});
+			} else {
+				throw new Error("Invalid field specification!");
+			}
+			return this;
 		},
-		secondItemNameValue: {
-			locateStrategy: 'xpath',
-			selector: '//tbody/tr[2]/td/a[contains(@class, "ItemList__value ItemList__value--text")][1]',
+
+		/**
+		 * Asserts that the specified item delete icon is visible.
+		 *
+		 * @param {Object} icons The delete icons specification.
+		 * @param {number} icon.row The one-based item row number of the delete icon in the list screen.
+		 * @param {number} icon.column The one-based column number of the delete icon in the list screen.
+		 */
+		assertItemDeleteIconVisible: function (icons) {
+			var browser = this;
+			if (icons) {
+				icons.forEach(function (icon) {
+					browser.expect.element(getItemDeleteIconSelector(icon)).to.be.visible;
+				});
+			} else {
+				throw new Error("Invalid icons specification!");
+			}
+			return this;
 		},
-	},
-	commands: [{
-		deleteItem: function (selector) {
-			return this
-				.click(selector);
+
+		/**
+		 * Click delete item icons.
+		 *
+		 * @param {Object} icons The delete item icons specification.
+		 * @param {number} icon.row The one-based item row number of the delete icon to click in the list screen.
+		 * @param {number} icon.column The one-based item column number of the delete icon to click in the list screen.
```

Comment:
Why are there rows here? Are the delete icons not always down the right-hand side of the page..?